### PR TITLE
Adding linux commandline cheatsheet

### PR DESCRIPTION
**Pull request by pecey**
Adding linux commandline cheatsheet

//cc @hemanth 

IA : https://duck.co/ia/view/linux_command_line_cheat_sheet

### CODE REVIEW

**Comment by daxtheduck** [comment]
@pecey Hey!
        
Thanks for taking the time to contribute! We really appreciate it.

We work closely with every contributor to make Instant Answers the best they can be, so we appreciate your patience as we look over your code. From here, the process usually goes like this:

1. Pull Request is reviewed by the DuckDuckGo staff and community
2. Staff and community will leave feedback with any necessary updates to the function or design.
3. Once you've made any necessary corrections, then your Instant Answer will be merged and deployed live on DuckDuckGo!

While the team and the community review your submission, we'd like to make sure you look good! If you'd like a free DuckDuckGo tshirt, please send over the following info to merch@duckduckgo.com, and be sure to put "Dev Merch" in the subject line:

Name:
Street Address:
City:
State / Province:
Country:
Phone (Non-US Orders only): 
T-shirt size:
GitHub username:
Link to this Pull Request:

If you have any questions along the way, feel free to ask them here. Our staff and community are also available on Slack to answer any questions you may have. If you'd like to join us there please send an email to <quackslack@duckduckgo.com>.

>More Info: https://duck.co/duckduckhack/submission_and_review

Thanks!

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

File: share/goodie/cheat_sheets/json/linux-commandline.json [hunk 1]
```diff
@@ -0,0 +1,92 @@
+{
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/linux-commandline.json]: @pecey Thanks for contributing and welcome to DuckDuckHack!

The filename needs to be `linux-command-line` to match the ID.

I believe we already have some Linux/Ubuntu cheat sheets though, so you may need to check if those cover all or most of this content

**Comment by moollaza** [comment]
@pecey congrats! Its live on beta! https://beta.duckduckgo.com/?q=linux+commandline+cheat+sheet&ia=cheatsheet&iax=1

Because of the filename it will only trigger on "linux commandline (cheat sheet | help | reference | etc)"

**Comment by pecey** [comment]
@moollaza Some help? The second pull request was meant for another goodie. Looks like I messed it up.

**Comment by pecey** [comment]
@moollaza : Should I close this PR and send a new PR? Had to delete the repo due to a mess up.

**Comment by hemanth** [comment]
@pecey You can fix this PR itself, just make the changes in your [fork](https://github.com/pecey/zeroclickinfo-goodies).

**Comment by pecey** [comment]
@hemanth : Can you help? I had to delete the previous repo, so I am not sure how to fix this up.

**Comment by hemanth** [comment]
Trying to reach you on slack.

**Comment by pecey** [comment]
A similar IA already exists. 

https://duck.co/ia/view/linux_cheat_sheet

So closing this one.

**Pull request closed without merging**